### DISCUSSION TITLE
Megathread: More documentation needed

### DISCUSSION THREAD

**Discussion by Mrtcndkn** [builttoroam/device_calendar#422] [General]
Even tho it's one of the more popular calendar repos out there for flutter, I find it hard to focus and get into the whole thing without proper examples. I think it would immensely improve the experience if we had examples for every method that we can use per event, which class involves them, what are they good for and so on. Otherwise, one needs to dive into the whole code base while all they want to do is just creating an event on the calendar. Thanks for this library, btw.

**Comment by thomassth** [comment]
This is indeed one of the areas to improve

Right now the example app is quite complete, and you may reference code from it if you need help
